tests/spec/core/xref-spec.js

```diff
@@ -721,6 +721,53 @@ describe("Core — xref", () => {
     });
   });
 
+
+  describe("Handle configurations correctly", () => {
+    it("xref as array of specifications", async () => {
+      const config = { xref: ["a", "b", "c"] }; 
+      const body = `
+      <section id='abstract'>
+        <p>This is required.</p>
+      </section>
+      <section id='sotd'>
+        <p>This is required.</p>
+      </section>`;
+      const ops = makeStandardOps(config, body);
+      const doc = await makeRSDoc(ops);
+      expect(doc.body.dataset.cite).toContain("a b c");
+    });
+    
+    it("xref as profile string - valid profile", async () => {
+      const config = { xref: "W3C" }; 
+      const body = `
+      <section id='abstract'>
+        <p>This is required.</p>
+      </section>
+      <section id='sotd'>
+        <p>This is required.</p>
+      </section>`;
+      const ops = makeStandardOps(config, body);
+      const doc = await makeRSDoc(ops);
+      expect(doc.body.dataset.cite).toContain("HTML INFRA URL PROMISES-GUIDE WEBIDL DOM FETCH");
```

Comment:
Maybe something like: 

```suggestion
      expect(doc.body.dataset.cite.split(" ")).toContain(["HTML", "INFRA", "URL", "PROMISES-GUIDE", "WEBIDL", "DOM", "FETCH");
```

Reply:
(you will need to check Jasmine docs for how to do the comparison between arrays..)

Reply:
Yeah, I found out how to check an array for a subarray. I will add it along with the wrong profile tests.